tensorboard/data/server/Cargo.toml

```diff
@@ -67,6 +68,9 @@ additional_flags = [
     "--cfg=has_std",
 ]
 
+[package.metadata.raze.crates.proc-macro-error.'1.0.4']
+gen_buildrs = true
+
```

Comment:
Is this change required since clap can emit error?

Reply:
Yes, I expect that the `clap` dep on `proc-macro-error` is to make it
easy for `clap` to report compile-time errors. This line is needed
because `proc-macro-error` uses a `build.rs` script to detect your Rust
version and enable certain features if they’re stable.